bindings/wasm/examples-account/src/memory_storage.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {ChainState, DID, Ed25519, Generation, IdentityState, KeyLocation, KeyPair, KeyType, MethodType, PrivateKey, Signature, Storage} from './../../node/identity_wasm.js';
+
+class MemStore implements Storage {
+    private _publishedGenerations: Map<DID, Generation>;
+    private _chainStates: Map<DID, ChainState>;
+    private _states: Map<DID, IdentityState>;
+    private _vaults: Map<DID, Map<KeyLocation, KeyPair>>;
+
+    constructor() {
+        this._publishedGenerations = new Map();
+        this._chainStates = new Map();
+        this._states = new Map();
+        this._vaults = new Map();
+    }
+
+    public async setPassword(_encryptionKey: Uint8Array): Promise<void> {}
+
+    public async flushChanges(): Promise<void> {}
```

Comment:
We might need a comment here explaining that these functions are not necessary here but are needed for other `Storage` implementations. I.e. it's fine to have no-op functions?

Aside: still not sure if we're going to keep `setPassword`, since I'm hoping that `Stronghold` will take over password management. Basically authentication should be part of the implementation rather than the interface, ideally.

Reply:
Fully agree `setPassword` should be removed, but perhaps in a follow-up PR? The changes required are minimal, I think, so it could be done here as well. I'm fine either way. Passwords should be passed to storage constructors and not be required post-creation anymore, which will, for example, be the case with the upcoming stronghold refactor. Some storages might not even need a password, so it shouldn't be part of the interface.

Reply:
I'm fine with any changes being added to an issue or appended to #669, as long as there's something to keep track of them.